scripts/build_test_configs.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: : 2022 PyPSA-Earth Authors
+#
+# coding: utf-8
+"""
+Write option files (configs) for the Continous Integration tests
+
+The config.tutorial.yaml has all options.
+The test/* config files have only key/value strings that are different from the tutorial config.
+The below scripts 'updates' the test configs and adds all options of the tutorial config.
+"""
+import collections.abc
+import copy
+import os
+from pathlib import Path
+
+from ruamel.yaml import YAML
+
+
+def update(d, u):
+    for k, v in u.items():
+        if isinstance(v, collections.abc.Mapping):
+            d[k] = update(d.get(k, {}), v)
+        else:
+            d[k] = v
+    return d
+
+
+if __name__ == "__main__":
+    if "snakemake" not in globals():
+        from _helpers import mock_snakemake
+
+        os.chdir(os.path.dirname(os.path.abspath(__file__)))
+        snakemake = mock_snakemake("build_test_configs")
+
+    # Input paths
+    fptutorial = Path(Path.cwd(), snakemake.input.tutorial)
+    fp0 = Path(Path.cwd(), snakemake.input.test_standard)
```

Comment:
See the previous comment.
In this file, the same code for fp0-3 is basically repeated. I think we could rewrite it using a loop.